static/css/app.css

```diff
@@ -442,4 +446,5 @@ input.toggle-round:checked + label:after {
 
 .flow {
     display: flow-root;
+    padding-bottom: 50px;
```

Comment:
Why is this padding necessary?

Reply:
Padding is required at the end of the page for mobile and other scroll screens.

Reply:
@canihavesomecoffee ??